apps/backend/src/authn/github.strategy.ts

```diff
@@ -0,0 +1,75 @@
+import {Injectable, UnauthorizedException} from '@nestjs/common';
+import {PassportStrategy} from '@nestjs/passport';
+import axios from 'axios';
+import {Strategy} from 'passport-github';
+import {ConfigService} from '../config/config.service';
+import {User} from '../users/user.model';
+import {AuthnService} from './authn.service';
+
+interface GithubProfile {
+  name: string;
+  [key: string]: any;
```

Comment:
Are these `[key: string]: any;` types actually necessary? I tried removing one of them locally and it seems that everything still compiles successfully.

Reply:
They don't seem to be necessary, will remove them.